### PR TITLE
Use external modularized pinning backends

### PR DESCRIPTION
**Pull request by ukstv**
Drop right-on dependency on Powergate client

### CODE REVIEW

File: packages/ceramic-core/package.json [hunk 1]
```diff
@@ -37,7 +37,9 @@
     "@ethersproject/base64": "^5.0.2",
     "@ethersproject/providers": "^5.0.0-beta.144",
     "@ethersproject/random": "^5.0.2",
-    "@textile/powergate-client": "^0.2.0",
+    "@pinning-aggregation/aggregation": "^2.1.1",
+    "@pinning-aggregation/common": "^2.0.0",
+    "@pinning-aggregation/ipfs-pinning": "^2.0.1",
```
Comment (oed) [inline packages/ceramic-core/package.json:42]: I guess it would be nice for these deps to be part of this monorepo as well?

**Review by oed — COMMENTED** [review]
Awesome! I'll let @simonovic86 provide a more in depth review 👍

File: packages/ceramic-core/package.json (hunk 1 above)
Comment (ukstv) [reply packages/ceramic-core/package.json:42]: Could be.

Comment (ukstv) [reply packages/ceramic-core/package.json]: They are the part of the repo now.

**Review by simonovic86 — APPROVED** [review]
sweet!

File: packages/ceramic-core/package.json (hunk 1 above)
Comment (oed) [reply packages/ceramic-core/package.json]: Very cool, thanks!

**Pull request merged**